installer/bundle/bundle_skel_Linux.sh

```diff
@@ -690,8 +569,8 @@ fi
 #
 
 EXIT_STATUS=0
-SCX_EXIT_STATUS=0
 OMI_EXIT_STATUS=0
+SCX_OMI_EXIT_STATUS=0
```

Comment:
Not quite sure why you renamed SCX_EXIT_STATUS to SCX_OMI_EXIT_STATUS. I suggest you keep it as `SCX_EXIT_STATUS` (consistent with other files, and clearer as well).